src/test/docker/docker-compose.yml

```diff
@@ -3,6 +3,11 @@ services:
   dropwizard:
     build: ./docker-dropwizard-example/
     container_name: dropwizard
+    image: example/dropwizard
```

Comment:
Using an image here names it in the local repo.  Doing this matches the image name given in the dockerBuildImage script.